client/containers/containers_defs.cpp

```diff
@@ -27,6 +27,16 @@ QString ContainerProps::containerToString(amnezia::DockerContainer c){
     return "amnezia-" + containerKey.toLower();
 }
 
+QString ContainerProps::containerTypeToString(amnezia::DockerContainer c){
```

Comment:
This function don't needed
You can use ContainerProps::defaultProtocol to get default Protocol for container, and ProtocolProps::protoToString if needed